image_enc_test.go

```diff
@@ -136,14 +133,8 @@ func TestImageEncryption(t *testing.T) {
 		},
 	}
 
-	l, err := ls.Create(ctx, leases.WithRandomID(), leases.WithExpiration(5*time.Minute))
```

Comment:
This test doesn't need a lease such as `WithLease`?

Reply:
Ok - i think i figured out what the issue was. I think it is a bad `gc` label.

Reply:
Pushing a fix 🤞 .